src/features/CareViewSummary/utils/CareViewSummary.js

```diff
@@ -63,3 +63,11 @@ export const getWardDetails = async () => {
   }
   return [];
 };
+
+export const getSlidesPerView = (isMobileView, isTabletView) => {
+  if (isMobileView) {
+    return 2;
+  } else if (isTabletView) {
+    return Math.floor((window.outerWidth * 0.9) / 142);
```

Comment:
what's this logic here? can you give the comment how are you calculating over here? like what is 142 etc

Reply:
For mobile View We are expecting it to show 2 tiles at a time.
This calculation is for the Tablet View to show as many tiles possible based the parent component size. From the Figma each tile is 142 px and the width of the parent component is 90% of the Screen so I did window.outerWidth *0.9 which the parent component's width and dividing by 142 gives the number of Tiles which can fit in the parent.

Reply:
can you add this info as comment before this logic like why 142 and 0.9?